src/lib/vasi-sync/Cargo.toml

```diff
@@ -10,6 +10,7 @@ num_enum = { version = "0.6.1", default-features=false }
 rustix = { version = "0.37.19", default-features = false, features=["fs", "thread", "process"] }
 static_assertions = "1.1.0"
 vasi = { path = "../vasi" }
+zwohash = { version = "0.1.2", default-features=false }
```

Comment:
Is there a reason for using zwohash rather than fxhash (rustc-hash)? We already depend on rustc-hash and it has a lot more users (40,226,224 downloads vs 7,210).

Reply:
fxhash isn't no_std :(

I don't have any particular attachment to zwohash though, if there's another no_std hash we ought to consider

Reply:
It looks like it is if you specify `default-features=false`?

https://github.com/rust-lang/rustc-hash/blob/5ff7e6ae50e770381cb7bb393bfd22e016f2b8a6/src/lib.rs#L26
https://github.com/rust-lang/rustc-hash/blob/5ff7e6ae50e770381cb7bb393bfd22e016f2b8a6/Cargo.toml#L13

Reply:
Ah, I was looking at the [fxhash](https://crates.io/crates/fxhash) crate. Sure, I'll switch it over.

Reply:
Sounds good, I don't have a preference either if there's an advantage using a different hash function, but all other things equal I think it would probably be best to choose the popular (and rust-lang-owned) one.